### ISSUE TITLE
Cannot set property 'subscriptions' of undefined (after updating to Atom 1.16.0-beta0)

### ISSUE THREAD

**Issue by kankaristo** [AtomLinter/linter-glsl#53]
Started getting the below error after updating to Atom 1.16.0-beta0. Tried reinstalling linter-glsl, but still getting the error.

Atom gives a "View Issue" link to https://github.com/AtomLinter/linter-glsl/issues/26, but that seems to unrelated.

**Atom**: 1.16.0-beta0 x64
**Electron**: 1.3.13
**OS**: Ubuntu 16.04.2
**Thrown From**: [linter-glsl](https://github.com/AtomLinter/linter-glsl) package 2.1.1

### Stack Trace

Failed to activate the linter-glsl package

```
At Cannot set property 'subscriptions' of undefined

TypeError: Cannot set property 'subscriptions' of undefined
    at Object.activate (linter-glsl.js:199:5)
    at Package.module.exports.Package.activateNow (/app.asar/src/package.js:218:19)
    at /app.asar/src/package.js:190:32
    at Package.module.exports.Package.measure (/app.asar/src/package.js:96:15)
    at /app.asar/src/package.js:183:26
    at Package.module.exports.Package.activate (/app.asar/src/package.js:180:34)
    at PackageManager.module.exports.PackageManager.activatePackage (/app.asar/src/package-manager.js:550:34)
    at /app.asar/src/package-manager.js:531:29
    at Config.module.exports.Config.transactAsync (/app.asar/src/config.js:337:18)
    at PackageManager.module.exports.PackageManager.activatePackages (/app.asar/src/package-manager.js:526:19)
    at PackageManager.module.exports.PackageManager.activate (/app.asar/src/package-manager.js:508:46)
    at /app.asar/src/atom-environment.js:826:28
```

### Commands

```
     -3:41.3.0 tree-view:show (atom-workspace.workspace.scrollbars-visible-always.theme-base16-tomorrow-dark-theme.theme-atom-dark-ui)
     -1:42.7.0 settings-view:view-installed-packages (atom-pane.pane.active)
  2x -1:26.6.0 core:confirm (input.hidden-input)
```

### Non-Core Packages

```
advanced-open-file 0.16.5 
arpeggio 1.1.2 
atom-alignment 0.12.1 
atom-debugger 0.1.6 
atom-fuzzy-grep 0.15.0 
atom-keyboard-macros 0.9.2 
auto-detect-indentation 1.3.0 
autocomplete-glsl 0.2.3 
autocomplete-lua 0.8.0 
autocomplete-paths 1.0.5 
autocomplete-python 1.8.60 
autocomplete-xml 0.9.4 
blame 0.10.2 
branch-status 0.8.3 
chary-tree-view 0.2.3 
clipboard-history 0.6.6 
color-tabs 0.1.8 
color-tabs-regex 0.2.5 
ctrl-dir-scroll 0.2.3 
external-open 0.6.3 
file-icons 2.0.17 
file-watcher 1.1.0 
flex-tool-bar 0.12.0 
foldername-tabs 0.2.4 
git-control 0.8.2 
git-log 0.4.1 
git-plus 7.3.3 
git-time-machine 1.5.6 
highlight-bad-chars 1.0.5 
highlight-line 0.12.0 
highlight-selected 0.12.0 
jumpy 3.1.3 
language-arduino 0.4.2 
language-brainfuck 0.2.0 
language-gitignore 0.3.0 
language-glsl 2.0.1 
language-groovy 0.6.0 
language-ini 1.18.0 
language-latex 1.0.0 
language-lisp 0.2.0 
language-log 1.8.0 
language-lua 0.9.11 
language-qml 1.0.0 
language-qtpro 0.6.0 
language-r 0.4.1 
language-restructuredtext 1.0.2 
language-svg 0.9.2 
language-swig-wrapper 0.2.0 
language-x86 1.0.1 
lcov-info 0.12.0 
line-diff-details 1.5.0 
line-jumper 0.14.1 
linter 1.11.23 
linter-clang 3.4.8 
linter-coffeelint 1.2.1 
linter-csslint 1.3.4 
linter-glsl 2.1.1 
linter-jshint 3.0.3 
linter-less 2.6.0 
linter-lua 1.0.2 
linter-luacheck 1.1.7 
linter-pylint 1.2.2 
linter-shellcheck 1.4.3 
linter-xmllint 1.4.1 
merge-conflicts 1.4.4 
minimap 4.26.8 
minimap-bookmarks 0.4.2 
minimap-find-and-replace 4.5.2 
minimap-git-diff 4.3.1 
minimap-highlight-selected 4.5.0 
minimap-linter 1.6.0 
minimap-selection 4.4.0 
multi-cursor-plus 1.2.0 
multi-wrap-guide 0.28.2 
pigments 0.39.0 
pinned-tabs 0.4.2 
pretty-json 1.6.1 
process-palette 0.12.0 
project-colorize 0.2.0 
project-manager 3.3.3 
revert-buffer 0.5.0 
scratch 1.2.0 
script 3.14.1 
sort-lines 0.14.0 
switch-header-source 0.22.0 
symbols-tree-view 0.13.2 
todo-show 1.11.0 
toggle-packages 1.1.0 
toggle-quotes 1.0.1 
tool-bar 1.0.1 
trailing-spaces 0.4.0 
tree-view-breadcrumb 0.8.1 
tree-view-open-files 0.3.0 
wakatime 6.0.13 
```

**Comment by Arcanemagus** [comment]
Hmmm, this looks like the same issue you were hitting @KC-SI.

I'm trying to get CI working again, then will be looking into this. I know v1.16.0 changed the transpilation Atom does so it's possible something broke here.

**Comment by Arcanemagus** [comment]
Fixed in v2.1.2! 🎉

**Comment by andystanton** [comment]
Amazing work @Arcanemagus!
